catalog/build/py/package/catalog_build/build.py

```diff
@@ -471,10 +639,18 @@ def build_files(
 
   if extract_primary_data:
     primarydata_df.to_csv(primary_output_path, index=False, sep="\t")
-
     print(f"Wrote to {primary_output_path}")
   
   if qc_report_path is not None:
     qc_report_text = make_qc_report(**qc_report_params)
     with open(qc_report_path, "w") as file:
       file.write(qc_report_text)
+
+  if len(taxonomic_levels_for_tree) > 0:
+    # Use the taxonomy IDs from the genomes_df to build the species tree
+    # Pass the previously fetched species_info to avoid another API call
+    species_tree = get_species_tree(genomes_df["taxonomyId"], taxonomic_levels_for_tree, species_info)
```

Comment:
```suggestion
    species_tree = get_species_tree(list(genomes_df["taxonomyId"]), taxonomic_levels_for_tree, species_info)
```

Keeping the taxonomy IDs as a Pandas series causes issues with the `in` operator down the line